dynamo_db_insert_block.py

```diff
@@ -4,13 +4,12 @@
 from boto.dynamodb2.table import Table
 
 from nio.properties import StringProperty, VersionProperty
-from nio.util.discovery import discoverable
+from nio.block.mixins.enrich.enrich_signals import EnrichSignals
 
 from .dynamo_db_base_block import DynamoDBBase
 
 
-@discoverable
-class DynamoDBInsert(DynamoDBBase):
+class DynamoDBInsert(EnrichSignals, DynamoDBBase):
```

Comment:
Why is this an EnrichSignals block? It extends the mixin but then doesn't use any of the mixin methods?